USE-CASES/govtech-health-monitoring.md

```diff
@@ -0,0 +1,117 @@
+## Title: Public Health Monitoring
+
+### Submitter(s): 
+
+Jennifer Lin
+
+### Reviewer(s):
+
+Michael McCool, Michael Lagally
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+<please leave blank>
+
+### Class: 
+
+<please leave blank>
+
+### Status: 
+
+<please leave blank>
+
+### Target Users
+
+Agencies, companies and other organizations in a Smart City with 
+significant foot traffic.
+
+### Motivation:
+
+A system to monitor the health of people in public places is useful to
+control the spread of infectious diseases.  In particular, we would like
+to identify individuals with temperatures outside the norm (i.e. running
+a fever) and then take appropriate action.
+
+This mechanism should be non-invasive and non-contact since the solution
+should not itself contribute to the spread of infectious diseases.
+
+### Expected Devices:
+
+One of the following:
+* A thermal camera.
+* Face detection (AI) service.
+
+Optional:
+* RGB and/or depth camera registered with the thermal camera
+* Some way to identify location (optional)
+Note that location might be static and configured during installation,
+but might also be based on a localization technology if the device needs to be 
+portable (for example, if it needs to be set up quickly for an event).
+
+### Expected Data:
+
+* Sensor ID
+* Timestamp
+* Number of people identified with a fever in image
+* Estimated temperature for each person
+   * May be coarse, low/normal/high
+* Location
+   * Latitude, Longitude, Altitude, Accuracy 
+   * Semantic (eg a particular building entrance)
+* Thermal image
+
+Optional:
+* RGB image
+* Depth image
+* Localization technology (see localization use case)
+* Integration with local IoT devices: gates, lights, or people (guards)
+* Bounding boxes around faces of identified people in image(s)
+
+Note: the system should be capable of notifying consumers,
```

Comment:
consumers is a bit ambiguous, what about security personnel?

Reply:
well, it could also be consumers, e.g. government statistics, actual consumers wanting to know if the mall is crowded, etc.  However... I have added ("such as security personnel") as an example.